app/views/components/_notice.html.erb

```diff
@@ -3,8 +3,15 @@
     description_text ||= ""
     description_govspeak ||= ""
   %>
+
   <section class="app-c-notice" aria-label="Notice">
-    <h2 class="app-c-notice__title"><%= title %></h2>
+
+    <% if description_text.present? || description_govspeak.present? %>
```

Comment:
My ruby experience may have grown since then. @fofr should be able to rule whether this suggestion is worth changing or not.